src/TweetPublisher/TweetPublisher.php

```diff
@@ -54,11 +54,17 @@ public function run(): void
 
         $tweetsToPublish = $this->excludeAlreadyPublishedTweets($allPostTweets, $allPublishedTweets);
         if (! count($tweetsToPublish)) {
+            $this->symfonyStyle->warning('There is no new tweet to publish. Add a new one to one of your post under "tweet:" option.');
```

Comment:
rename symfonyStyle to logger  - imho zde není podstatné, jakým způsobem logger funguje, že má něco společného se symfony